front/src/components/ArticlePreview.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useRef, useEffect } from 'react'
-import { useSelector } from 'react-redux'
 import { useParams } from 'react-router-dom'
+import { applicationConfig } from '../stores/applicationConfig.jsx'
```

Comment:
Dans ce cas je verrais plutôt un truc du genre :

```suggestion
import { applicationConfig } from '../stores/applicationConfig.jsx'
import { backendEndpoint } from '../config.js'
```

Reply:
Tu as raison 👍🏻 
On garde `applicationfConfig` ou c'est redondant "application" ?

Reply:
Dans le nom du fichier ? J'ai pas d'avis :D